samples/meeting-stage-reaction-raisehand-api/csharp/README.md

```diff
@@ -0,0 +1,200 @@
+---
+page_type: sample
```

Comment:
Add a link from root readme file.
Also, make sure to raise a documentation bug to link this sample once this sample is publicly available.